src/dsa/rpo_falcon512/keys/public_key.rs

```diff
@@ -0,0 +1,135 @@
+use super::{
+    super::{Rpo256, LOG_N, N, PK_LEN},
+    ByteReader, ByteWriter, Deserializable, DeserializationError, FalconFelt, Felt, Polynomial,
+    Serializable, Signature, Word, MODULUS,
+};
+use alloc::string::ToString;
+use core::ops::Deref;
+use num::Zero;
+
+// PUBLIC KEY
+// ================================================================================================
+
+/// A public key for verifying signatures.
+///
+/// The public key is a [Word] (i.e., 4 field elements) that is the hash of the coefficients of
+/// the polynomial representing the raw bytes of the expanded public key.
+///
+/// For Falcon-512, the first byte of the expanded public key is always equal to log2(512) i.e., 9.
```

Comment:
I'm not sure this line is relevant in this place any more. Maybe it should be a comment somewhere in serializaiton/deserialization code?

Reply:
Removed as the relevant place in the serialization code already has a much better descriptive comment